libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,7 +1,7 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
   <div class="reading-list-item" *ngFor="let b of readingList$ | async">
     <div>
-      <img class="reading-list-item--cover" [src]="b.coverUrl" />
+      <img class="reading-list-item--cover" [src]="b.coverUrl" alt="Book Cover"/>
```

Comment:
Again here, check the decision tree linked in another comment.

Reply:
Made the appropriate change